mod/files/scripts/utils.lua

```diff
@@ -421,5 +422,6 @@ function serialize_data()
         table.insert(serialized, { stats, always_cast, deck })
     end
 
-    return json.encode({ wands = serialized, inventory = inventory, items = items, progress = progress, version = version, info = info })
+    return json.encode({ wands = serialized, inventory = inventory, items = items, progress = progress, version = version, info =
+    info })
```

Comment:
I don't think LUA is newline-sensitive, but is this a typo?
```suggestion
    return json.encode({ wands = serialized, inventory = inventory, items = items, progress = progress, version = version, info = info })
```